spec/models/impact_story_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe ImpactStory, type: :model do
+  it "returns the first part of the content according to the given limit" do
+    impact_story =
+      create(:impact_story, title: "For Testing", content: "Making some long content that needs to be shortened according to a given limit")
```

Comment:
Let's use https://github.com/faker-ruby/faker to generate randomized data for tests. This is a good idea because sometimes tests pass because it relies on hardcoded data. Using randomized data helps avoid accidentally coupling the data with the actual implementation you are testing.